### PR TITLE
A PR for Contact form where we set up the contact 

### PR DESCRIPTION
**Pull request by KALUNGI-J-BOBSON**
Added the form section
Added the MIT license
Setup linters for the index.html  and style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -250,26 +250,23 @@ <h2>
...
-            feature that you need built or a project that needs coding. I’d love
+              If you have an application you are interested in developing, 
+            features that you need built or a project that needs coding. I’d love
             to help with it.
             </h2>
           </div>
-        <form     
+          <form     
           action="https://formspree.io/f/xpznodvr"
           method="post"
```
Comment (ghost) [inline index.html:271]: - [ ] Kindly adjust the padding of your input elements to match the original design.

image


### Checkout original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Form submission working properly 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by enniopo** [comment]
Hi @KALUNGI-J-BOBSON - please go ahead and submit another review request when you are done implementing the changes requested by the CR. Thank you, and sorry for the inconvenience.

**Comment by KALUNGI-J-BOBSON** [comment]
I have implemented the changes ambut i dont have a review button i have
only the submit button

**Comment by KALUNGI-J-BOBSON** [comment]
Hello
Thanks
Iam trying to request another review having finished implementation of what
the previous CR asked me
But i see it tell me “You have no review requests to rate for this activity”

What can I do?

**Comment by KALUNGI-J-BOBSON** [comment]
Hello,i hope you are doing well,
I am trying to request for a review on my project but it shows me “you have
no review requests to rate for this activity “
Kindly help me out thanks

**Comment by enniopo** [comment]
@KALUNGI-J-BOBSON let me check this

**Comment by enniopo** [comment]
@KALUNGI-J-BOBSON you should be able to request a review now.

**Comment by KALUNGI-J-BOBSON** [comment]
Hello
I have just tried now after this mail
I stil can’t request for a review

**Comment by KALUNGI-J-BOBSON** [comment]
Thanks it had come now
Iam able

**Review by Olamarx — APPROVED** [review]
Hi @KALUNGI-J-BOBSON,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Contact form 👍🏼 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**